qa/tasks/cephfs/test_misc.py

```diff
@@ -10,6 +10,7 @@
 
 class TestMisc(CephFSTestCase):
     CLIENTS_REQUIRED = 2
+    MDSS_REQUIRED = 2
```

Comment:
Is there are way that number of MDSs can be changed for only one test case rather than the entire group of test cases? Because this changes the environment for all the test cases which is incorrect in my opinion.

But, on other hand, it shouldn't affect the other test cases since the extra MDS(s) won't be active on their filesystem(s) until max_mds is explicitly set to 2